src/ui/public/agg_types/param_types/filter/field_filters.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { FieldParamType } from '../';
+import { AggType } from '../..';
+import { IndexPattern } from '../../../index_patterns';
+import { AggConfig } from '../../../vis';
+
+type AggTypeFieldFilter = (
+  fields: any[],
+  aggType: AggType,
```

Comment:
I think we should not pass in `aggType` directly to the filters. `aggConfig.type` will always retrieve the type and we have no ambitions to remove that, since `aggConfig` is pretty much just one specifically configered aggregation that always will have a type. So I think we should rather reduce this to just pass in `aggConfig` here.

Reply:
removed references to `aggType`